### PR TITLE
Emmisary presubmit

### PR DESCRIPTION
**Pull request by jonahjon**
Signed-off-by: jonahjon <jonahjones094@gmail.com>

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: jobs/aws/eks-anywhere-build-tooling/emmisary-presubmitsyaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+presubmits:
+  aws/eks-anywhere-build-tooling:
+  - name: emmisary-tooling-presubmit
+    always_run: false
+    run_if_changed: "EKS_DISTRO_BASE_TAG_FILE|^build/lib/.*|Common.mk|projects/emmisary-ingress/emmisary/.*"
```
Comment (jaxesn) [inline jobs/aws/eks-anywhere-build-tooling/emmisary-presubmitsyaml:19]: This tag file will be the python one

**Review by jaxesn — CHANGES_REQUESTED** [review]

File: jobs/aws/eks-anywhere-build-tooling/emmisary-presubmitsyaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        - -c
+        - >
+          trap 'touch /status/done && rm -rf /home/prow/go/src/' EXIT
+          &&
+          build/lib/buildkit_check.sh
+          &&
+          if $(make check-project-path-exists); then make build -C $PROJECT_PATH; fi
+        env:
+        - name: PROJECT_PATH
+          value: projects/emmisary/emmisary
```
Comment (abhay-krishna) [inline jobs/aws/eks-anywhere-build-tooling/emmisary-presubmitsyaml:47]:
```suggestion
          value: projects/emissary-ingress/emissary
```

**Comment by abhay-krishna** [comment]
`emmisary` -> `emissary` across the job definition and filename.

**Comment by jonahjon** [comment]
> emissary

o yea good catch

**Review by abhay-krishna — APPROVED** [review]
/lgtm

**Review by jaxesn — APPROVED** [review]
/approve

**Pull request merged**